{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:94:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:43:26.148Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:94:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:44:39.037Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:94:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:45:35.833Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:96:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:46:40.116Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:96:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:47:17.671Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:96:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:48:02.834Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:96:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:48:29.509Z"}
{"level":"error","message":"Error signing in: Error signing in: Invalid login credentials","method":"POST","path":"/api/users/supabase/signin","stack":"Error: Error signing in: Error signing in: Invalid login credentials\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:96:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:55:10.733Z"}
{"level":"error","message":"Error upserting events: insert or update on table \"Event\" violates foreign key constraint \"Event_userId_fkey\"","method":"POST","path":"/api/containers/updateStatus","stack":"Error: Error upserting events: insert or update on table \"Event\" violates foreign key constraint \"Event_userId_fkey\"\n    at C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:50:11\n    at Generator.next (<anonymous>)\n    at fulfilled (C:\\Users\\yosho\\Documents\\Panchito\\Tracking-CTEnvios\\backend-typescript\\src\\databases\\supabase\\supabase_db.ts:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-11-08T18:58:25.897Z"}
